ci/scale-test.sh

```diff
@@ -30,6 +30,7 @@ $OVNSUDO docker exec ovn-rally rally task report $TASKID --out /root/create-netw
 $OVNSUDO docker exec ovn-rally rally task results $TASKID > ./create-networks-data.json
 $OVNSUDO docker cp ovn-rally:/root/create-networks-output.html .
 $OVNSUDO docker exec ovn-rally rally task delete --uuid $TASKID
+$OVNSUDO rm -rf /tmp/rally-ovs-output.raw
```

Comment:
Just for my clarification, are these part of this change or another issue you found? I'm fine leaving them in as part of this commit if it's easier, just wasn't sure and wanted to clarify.